### PR TITLE
[DEV APPROVED] 8056_Endofyear_Warning

### PR DESCRIPTION
**Pull request by thecountgs**
We did similar work last year and I was able to re-use code to deploy this end of year warning for the Redundancy Pay Calculator.

Previous code used for reference:
https://github.com/moneyadviceservice/frontend/commit/9a5f8ad9a037ecb5ce9c4e51a5efb1ef1e2036dc

|Welsh|English|
|------|-------|
|image|image|

---
This change is [Reviewable](https://reviewable.io/reviews/moneyadviceservice/frontend/1705)

### CODE REVIEW

**Comment by johnplayer1982** [comment]
Looks good! :+1:

**Comment by thecountgs** [comment]
The date has been confirmed and the warning has been scheduled accordingly.

**Comment by johnplayer1982** [comment]
:+1:

**Comment by tomas-stefano** [comment]
Good, very good 👍  👍 But can you add a tests to the Budget Announcement class? Like this: https://github.com/moneyadviceservice/frontend/commit/9a5f8ad9a037ecb5ce9c4e51a5efb1ef1e2036dc#diff-00d79349241ce1be3f8453cb5542bc59

**Comment by tomas-stefano** [comment]
Sorry, just testing the Github PR's feature :D

**Comment by thecountgs** [comment]
Tomas, I added rspec tests to the budget warning as requested.

File: features/add_warning_to_tools.feature [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  The following tool should have the warning if it has not been updated by March 16th. Redundancy Advice Tool - https://www.moneyadviceservice.org.uk/en/tools/redundancy-pay-calculator.
+
+  Scenario Outline: before the budget announcement
+    Given Today's date is "5th March 2017"
+    When I visit the Redundancy Advice Tool in "<language>"
+    Then I should not see the warning about the impending changes in "<language>"
+
+    Examples:
+      | language |
+      | english  |
```
Comment (johnplayer1982) [inline features/add_warning_to_tools.feature]: @thecountgs These should be capitalised (English, Welsh), I think these tests will fail otherwise?

**Comment by Tr4pSt3R** [comment]
👍 👍

**Pull request merged**